### ISSUE TITLE
[BUG] Spring Boot Actuator health check returning 503 status for servers with org.jobrunr.background-job-server.enabled = false

### ISSUE THREAD

**Issue by anuragbnrj5** [jobrunr/jobrunr#343]
**Are you running the latest version of JobRunr?**
Yes / No
If no, please first upgrade to the latest JobRunr version and check whether the issue still exists.
On production, we are using 4.0.0.
But, I have tried using version 4.0.6 on my local and the issue is still present. 

**Describe the bug**
The deployment pipeline in my organization extensively uses actuator health checks (200 status) for determining the status of containers. When I am setting the `org.jobrunr.background-job-server.enabled = false`, in order to prevent the background job server from running on the same container as the request server, the health check returns 503. This is causing issues in the deployment.

**Environment**
I'm using JobRunr version: 4.0.0
I'm running on JRE / JDK (e.g. OpenJDK 1.8.0_292): OpenJDK 11
I'm using the following StorageProvider (e.g. Oracle / MongoDB / ...): Postgres 

**How to Reproduce**
Provide a simple Github project or code fragment which shows how to reproduce the issue (ideally a Github repo).
```text
Step 1: Set org.jobrunr.background-job-server.enabled=false in application.properties / application.yml
Step 2: Hit the health check endpoint for the application (/actuator/health)
Step 3: Check the response code. It is 503 (out of service).
```

**Expected behavior**
There should be some config properties to disable jobrunr health check for servers with` org.jobrunr.background-job-server.enabled=false`

**Additional context**
This is the response for the health check endpoint with components enabled
```json
{
    "status": "UP",
    "components": {
        "": {
            "status": "OUT_OF_SERVICE"
        },
        "db": {
            "status": "UP"
        },
        "diskSpace": {
            "status": "UP"
        },
        "ping": {
            "status": "UP"
        }
    }
}
```
The HealthIndicator bean for jobrunr seems to be missing a name because of which it cannot be disabled from the properties file for actuator healthcheck.

**Comment by rdehuyss** [comment]
Although technically correct (`OUT_OF_SERVICE != DOWN`), I agree that JobRunr should be as easy to use as possible. From JobRunr 5.0 it will return UP.

To solve your deployment issue now, read up on the Spring docs on how to return http 200.

**Comment by rdehuyss** [comment]
Solved in v5.0.0 branch.

**Comment by anuragbnrj5** [comment]
@rdehuyss Thank you for the quick fix.

For folks who need to return 200 status in actuator health checks, you can add `management.health.jobrunr.enabled=false` in the properties file.
